api/agent/protocol/json.go

```diff
@@ -142,7 +142,7 @@ func (h *JSONProtocol) writeJSONToContainer(ci CallInfo) error {
 	err = writeString(err, h.in, "}")
 
 	// close
-	err = writeString(err, h.in, "}\n\n")
+	err = writeString(err, h.in, "}\n")
```

Comment:
I still think writing `\n` in the end is not necessary. The result of `writeString` would be the same as you expect to get with `}\n`
If you call `writeString(err, h.it, "}")` it will look like:
```
}

```
on the next line new request data will appear, no additional padding really needed. The thing we actually get from this padding (line 76 and 145) is visibility for debugging purpose, nothing else. At this particular moment, there's no problem with distinguishing two JSON requests without padding. So, i still feel uncomfortable with padding (+2 bytes, but who really counts ;-)  )